zavod/zavod/helpers/names.py

```diff
@@ -326,36 +341,15 @@ def review_names(
         context: The current context.
         entity: The entity to apply names to.
         string: The raw name(s) string.
-        alias: If this is known to be an alias and not a primary name.
-        lang: The language of the name, if known.
     """
-    strings = [s for s in strings if s]
-
-    if not strings:
+    non_blank_strings = [s for s in strings if s]
+    if not non_blank_strings:
         return None
 
-    if settings.CI or not any(is_name_irregular(entity, s) for s in strings):
-        prop = "alias" if alias else "name"
-        for string in strings:
-            entity.add(prop, string, lang=lang)
+    if settings.CI or not any(is_name_irregular(entity, s) for s in non_blank_strings):
         return None
 
-    non_blank_strings = [s for s in strings if s and s.strip()]
-    raw_names = RawNames(entity_schema=entity.schema.name, strings=non_blank_strings)
-    names = clean_names(context, raw_names)
-
-    source_value = JSONSourceValue(
-        key_parts=[entity.schema.name] + non_blank_strings,
-        label="names",
-        data=raw_names.model_dump(),
-    )
-    review = review_extraction(
-        context,
-        source_value=source_value,
-        original_extraction=names,
-        origin=LLM_MODEL_VERSION,
-    )
-    return review
+    return _review_names(context, entity, non_blank_strings)
 
 
 def apply_reviewed_names(
```

Comment:
So IIUC, usually we'd use `apply_reviewed_names`, but if we only want to create the reviews, we use `review_names?`

I think I sort of liked the design (if I'm getting this correctly) where the return value of `review_names` would be fed to an `apply_names` function - thinking ahead, there might be some policy decisions we want to put into the `apply` function arguments (like what happens with new, unreviewed names maybe?). But I think this works, we can always evolve it.